pyrasterframes/src/main/python/docs/nodata-handling.pymd

```diff
@@ -1,2 +1,158 @@
 # "NoData" Handling
 
+## What is NoData?
+
+In raster operations, the preservation and correct processing of missing observations is very important. In [most dataframes and scientific computing](https://www.oreilly.com/learning/handling-missing-data), the idea of missing data is expressed as a null or NaN value. A great deal of raster data is stored for space efficiency. This typically leads to use of integral values and a "sentinel" value to represent missing observations. This sentinel value varies across data products and is usually called the NoData value. 
+
+RasterFrames provides a variety of functions to inspect and manage NoData within `tile`s. 
+
+## Cell Types
+
+To understand how NoData is handled in RasterFrames, we first need to understand the different underlying types of data called cell types. The cell types are GeoTrellis `CellType`s, so the [GeoTrellis documentation](https://geotrellis.readthedocs.io/en/latest/guide/core-concepts.html?#working-with-cell-values) is a valuable resource on how these are defined.
+
+Use the function `rf_cell_type` to find the cell type of a specific set of raster data.
+
+```python setup, echo=False
+import pyrasterframes
+from pyrasterframes.rasterfunctions import *
+import pyrasterframes.rf_ipython
+from IPython.display import display
+
+spark = pyrasterframes.get_spark_session()
+```
+
+```python ct_from_sen
+spark.read.raster('https://s22s-test-geotiffs.s3.amazonaws.com/luray_snp/B02.tif') \
+    .select(rf_cell_type('proj_raster')).distinct().show()
+```
+
+The function `rf_cell_types` provides a convenient list of basic cell types. Note that this list is not exhaustive. 
+
+```python
+rf_cell_types()
```

Comment:
Let's strike this. Maybe replace with the explicit list of "constants" in `CellType`.

Reply:
will do